addon/mongocrypt.cc

```diff
@@ -571,6 +571,11 @@ MongoCrypt::MongoCrypt(const CallbackInfo& info) : ObjectWrap(info) {
         mongocrypt_setopt_bypass_query_analysis(mongo_crypt());
     }
 
+    if (options.Has("keyExpirationMS")) {
+        mongocrypt_setopt_key_expiration(mongo_crypt(),
```

Comment:
The error thrown from libmongocrypt is a bit misleading when the number is negative:

```
expiration time must be less than 9223372036854775807, but got 18446744073708551616
```

An int64 will always fit in a uint64 if it is positive, but if it is negative the number wraps.  What would you think of checking the int64 value and asserting it isn't negative to provide a better error message in the bindings?  Something like `keyExpirationMS must be >= 0, received <>".

Reply:
Updated to throw if the value is negative.